build.gradle

```diff
@@ -100,7 +100,7 @@ allprojects {
 //    apply plugin: 'eclipse'
     apply plugin: 'idea'
 
-    version = '5.2.5-SNAPSHOT'
```

Comment:
why can't the PR just change code related to the title of the PR - all other stuff must be reverted